usr/src/cmd/zdb/zdb.c

```diff
@@ -94,6 +95,7 @@ int zfs_vdev_async_read_max_active;
 
 static const char cmdname[] = "zdb";
 uint8_t dump_opt[256];
+extern int aok;
```

Comment:
I'm not sure if this is _correct_, but I was able to silence the lint issue by changing this line like so:
```
diff --git a/usr/src/cmd/zdb/zdb.c b/usr/src/cmd/zdb/zdb.c
index e5797ae29d..3df7180c18 100644
--- a/usr/src/cmd/zdb/zdb.c
+++ b/usr/src/cmd/zdb/zdb.c
@@ -95,7 +95,7 @@ int zfs_vdev_async_read_max_active;

 static const char cmdname[] = "zdb";
 uint8_t dump_opt[256];
-extern int aok;
+volatile int aok;

 typedef void object_viewer_t(objset_t *, uint64_t, void *data, size_t size);
```

Reply:
Just out of curiosity... what was the lint error?

Reply:
```
==== lint warnings src ====

"/opt/jenkins/root/workspace/openzfs_openzfs_PR-494-64XNXXNYJTBMFXZULXEELHIF226JBJ6ZVXFRFO6EYEWBUT4REFUQ/usr/src/cmd/zdb/zdb.c", line 4152: warning: name used but not defined: aok in zdb.c(4152) (E_NAME_USED_NOT_DEF2)
```
you can see it in the build output for the automated build, [here](http://jenkins.open-zfs.org/blue/rest/organizations/jenkins/pipelines/openzfs/openzfs/branches/PR-494/runs/3/nodes/102/log/?start=0).

Reply:
I am going to say that that's bogus.  Either that or we aren't linking in libc (which provides aok) which building the lint library, but that seems unlikely.

Reply:
lint can be a pain, but I think we'll need to workaround this even if the message is bogus. `volatile` is used instead of `extern` in the `libfakekernel` code, so I think it'd be reasonable to do that here?